dataplane/src/main.rs

```diff
@@ -105,16 +111,42 @@ fn main() {
         info!("Starting RTE Worker on {lcore_id:?}");
         let rx_queue = devices[0].rx_queue(RxQueueIndex(i as u16)).unwrap();
         let tx_queue = devices[0].tx_queue(TxQueueIndex(i as u16)).unwrap();
-        WorkerThread::launch(lcore_id, move || loop {
-            let mut pkts: Vec<_> = rx_queue.receive().collect();
-            for pkt in pkts.iter_mut() {
-                let Ok((packet, _rest)) = Packet::parse(pkt.raw_data_mut()) else {
-                    info!("failed to parse packet");
-                    continue;
-                };
-                info!("received packet: {packet:?}");
+        WorkerThread::launch(lcore_id, move || {
+            // FIXME(mvachhar) construct pipline elsewhere, ideally from config file
+            let mut pipeline = Pipeline::new();
+            let config = Config::new();
```

Comment:
Even if not using a config file, we should probably move this to a dedicated function.

See also #225

Reply:
Done.